components/AndrewFace.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-param-reassign */
+import React, { Component } from 'react';
+import Sketch from 'react-p5';
+
+class AndrewFaceAnimation {
+  constructor(p5, x, y) {
+    this.p5 = p5;
+    this.x = x;
+    this.y = y;
+    this.img = p5.createImg("/img/andrew.png");
+  }
+
+  move() {
+    var xOffset = this.p5.floor(this.p5.random(0, 2));
+    var yOffset = this.p5.floor(this.p5.random(0, 2));
+
+    var newPositionX = this.x + xOffset;
+    var newPositionY = this.y + yOffset;
+
+    this.img.size(40, 45);
+    this.img.position(newPositionX, newPositionY);
+    this.img.style("z-index", -100);
```

Comment:
out of curiosity, could we set the size + z-index in a function that's not getting called repeatedly?

Reply:
Good catch! Updating